misc/skywater-pdk/build-lib.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I don't think we should do this here.

If we want the skywater-pdk to do an install step, then we should land that in the skywater-pdk repository.

Reply:
> I don't think we should do this here.

note: that's not actually done here, that's from the parent PR: https://github.com/hdl/conda-eda/pull/161

> If we want the skywater-pdk to do an install step, then we should land that in the skywater-pdk repository.

ah I see, note the "install" step is currently mainly used for `open_pdks` to depends on the `skywater-pdk` cell libraries packages at build time and be able to find them in a predicable place, they don't actually end up in the user system after doing `conda install open_pdks.sky130a`, if we don't foresee any other tools having to do that, an alternative could be to perform this as part of the `build.sh` of the `open_pdks` package itself (similar to how they document it in http://opencircuitdesign.com/open_pdks/install.html) and get rid of the skywater-pdk packages.

Reply:
Done